pilz_robot_programming/src/pilz_robot_programming/commands.py

```diff
@@ -320,7 +317,7 @@ class Ptp(_BaseCmd):
 
             acc_scale = vel_scale * vel_scale
     """
-    def __init__(self, vel_scale=_MAX_VEL_SCALE, acc_scale=None, *args, **kwargs):
+    def __init__(self, vel_scale=_DEFAULT_VEL_SCALE, acc_scale=None, *args, **kwargs):
```

Comment:
Please do not change to such a low default velocity. Choose a "reasonable" default velocity for PTP. This low default value was mainly intended for lin, circ and blending.

Reply:
@jschleicher

Reply:
Renamed the variable without changing its behaviour. Having different scales still seems like a typo to me; resolved into a new issue (related to operation modes?).

Reply:
Resolved into #191.